content/sensu-go/5.0/reference/entities.md

```diff
@@ -32,7 +32,7 @@ In addition, an entity can contain system information such as the hostname, OS,
 
 ## Proxy entities
 
-Proxy entities (formerly known as proxy clients, "Just-in-time" or "JIT" clients) are dynamically created entities, added to the entity store if an entity does not already exist for a check result. Proxy entity registration differs from keepalive-based registration because the registration event happens while processing a check result (not a keepalive message). Sensu proxy entities allow Sensu to monitor external resources on systems and/or devices where a sensu-agent cannot be installed (such a network switch) using the defined check ProxyEntityName to create a proxy entity for the external resource. Once created, proxy entities work much in the same way as any other Sensu entity.
+Proxy entities (formerly known as proxy clients, "Just-in-time" or "JIT" clients) are dynamically created entities, added to the entity store if an entity does not already exist for a check result. Proxy entity registration differs from keepalive-based registration because the registration event happens while processing a check result (not a keepalive message). Sensu proxy entities allow Sensu to monitor external resources on systems and/or devices where a sensu-agent cannot be installed (such a network switch) using the defined check ProxyEntityName to create a proxy entity for the external resource. Due to proxy entities not having keepalive messages, they will not work with the `deregister` attribute.
```

Comment:
I'd recommend leaving out this last sentence. The deregister attribute isn't the only difference with proxy entities, and calling it out prominently here creates that implication. I think a better home for information about individual attribute compatibility with proxy entities would be in the individual attribute tables.

Reply:
Yup, agreed. Removed. Might be worth a discovery of all attributes that may differ in another issue or PR, or even this one if that's in scope.